data/hai/unfettered side missions.txt

```diff
@@ -1426,3 +1427,132 @@ mission "Unfettered: Ambush"
 	on enter
 		# SPOILER: he doesn't want them to get destroyed by the Hai because that would damage Alpha-Unfettered reputations, also he takes it as a challenge to his strength (except he's on your ship)
 		dialog `Upon entering the system you see multiple Cerberus ships, ready to attack you. Choot'k sends a message to the Hai. "Do not interfere, it would damage both my honor and <last>'s!" He did not ask for your opinion, but thankfully you have seen worse than a bunch of pirates.`
+
+
+
+mission "Unfettered: Hunger Stories 1"
+	name `Transport family to <destination>`
+	description `Transport a bereaved Unfettered family to <destination>.`
+	source "Firelode"
+	destination "Hai-home"
```

Comment:
```suggestion
	destination "Hai-home"
	passengers 2
```
you forgot the family! also we usually add a blocked message but here I dont think we need to since its not a main story?